dataactvalidator/config/sqlrules/b11_award_financial_1.sql

```diff
@@ -4,4 +4,5 @@ SELECT
 FROM award_financial AS af
 WHERE af.submission_id = {}
 AND af.object_class NOT IN (SELECT object_class_code FROM object_class)
-AND af.object_class <> '000'
+AND af.object_class <> '0'
+AND af.object_class <> '00'
```

Comment:
We still want to look for `af.object_class <> '000'` since it's the same warning as before